iphone/Classes/TiUIWebView.h

```diff
@@ -6,51 +6,34 @@
  */
 #ifdef USE_TI_UIWEBVIEW
 
+#import "TiDimension.h"
```

Comment:
General question here: Did you copy the changes into this file (replaced) or did you copy the Ti.WKWebView  classes over and renamed them? I'd highly prefer the latter option to not mix up API's.